fission/main.go

```diff
@@ -174,14 +158,16 @@ func main() {
 
 	// timetriggers
 	ttNameFlag := cli.StringFlag{Name: "name", Usage: "Time Trigger name"}
-	ttCronFlag := cli.StringFlag{Name: "cron", Usage: "Time Trigger cron spec ('0 30 * * *', '@every 5m', '@hourly')"}
+	ttCronFlag := cli.StringFlag{Name: "cron", Usage: "Time Trigger cron spec ('0 30 * * * *', '@every 5m', '@hourly')"}
```

Comment:
Hey @life1347 I realised that 6th * placeholder is for seconds (Usually traditional standard syntax has been 5 placeholders for Minute, Hour, Day of Month, Month and Day of Week). So adding a quick explanation/tip might be helpful.